### PR TITLE
further style tweaks, and integrate quran-components

### PR DESCRIPTION
**Pull request by sabeurthabti**

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:32821

File: src/components/Nav/index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ export default ({ section, sections, handleSelect }) => {
           key={currentSection.id}
           onClick={() => handleSelect(currentSection.id)}
         >
-          {currentSection.name}
+          <a>{currentSection.name}</a>
```
Comment (mmahalwy) [inline src/components/Nav/index.js]: `<a tabIndex="-1">...`

**Comment by sabeurthabti** [comment]
@mmahalwy bro what do I need to do to ensure my tabs and spaces are consistent ? What do you use?

**Comment by mmahalwy** [comment]
I personally use atom and it's a life changer / memory hog :)

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:32861

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:32862

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:32863

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:32864

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:32865

**Comment by ahmedre** [comment]
Deployed to: http://dev.quranicaudio.com:32866

**Pull request merged**